src/hooks/hooks/handlers/BigOrderIsProfitHookHandler.ts

```diff
@@ -0,0 +1,139 @@
+        config.chains[giveChainId]!.fulfullProvider.connection;
+      const [giveUsdPrice, giveDecimals] = await Promise.all([
+        config.tokenPriceService.getPrice(
+          giveChainId,
+          order!.give.tokenAddress,
+          { logger: createClientLogger(arg.context.logger) }
+        ),
+        config.client.getDecimals(
+          giveChainId,
+          order!.give.tokenAddress,
+          giveConnection as Web3
+        ),
+      ]);
+
+      const giveUsdAmount = new BigNumber(order!.give.amount.toString())
+        .multipliedBy(giveUsdPrice)
+        .div(new BigNumber(10, giveDecimals))
+        .toNumber();
+
+      if (giveUsdAmount > this.minUsdAmount) {
+        logger.debug(
+          `Order give amount is big than set value(${giveUsdAmount} > ${this.minUsdAmount})`
+        );
+        const takeChainId = order!.take!.chainId;
+        const takeConnection =
+          config.chains[takeChainId]!.fulfullProvider.connection;
+        const [
+          takeDecimals,
+          reserveDecimals,
+          giveTokenSymbol,
+          takeTokenSymbol,
+          reservedTokenSymbol,
+        ] = await Promise.all([
+          config.client.getDecimals(
+            takeChainId,
+            order!.take.tokenAddress,
+            takeConnection as Web3
+          ),
+          config.client.getDecimals(takeChainId, arg.estimation.reserveToken),
+          config.client.getTokenSymbol(
+            giveChainId,
+            order!.give.tokenAddress,
+            giveConnection as Web3
+          ),
+          config.client.getTokenSymbol(
+            takeChainId,
+            order!.take.tokenAddress,
+            takeConnection as Web3
+          ),
+          config.client.getTokenSymbol(
+            takeChainId,
+            arg.estimation.reserveToken,
+            takeConnection as Web3
+          ),
+        ]);
+
+        const giveAmountWithoutDecimals = new BigNumber(
+          order!.give.amount.toString()
+        )
+          .div(new BigNumber(10, giveDecimals))
+          .toNumber();
+        const takeAmountWithoutDecimals = new BigNumber(
+          order!.take.amount.toString()
+        )
+          .div(new BigNumber(10, takeDecimals))
+          .toNumber();
+        const reservedAmountWithoutDecimals = new BigNumber(
+          arg.estimation.requiredReserveAmount.toString()
+        )
+          .div(new BigNumber(10, reserveDecimals))
+          .toNumber();
+
+        const message = `Order ${
+          arg.order.orderId
+        }: gives ${giveAmountWithoutDecimals} ${giveTokenSymbol} on ${giveChainId}, takes ${takeAmountWithoutDecimals} ${takeTokenSymbol} on ${takeChainId}}. Provide ${reservedAmountWithoutDecimals} ${{
+          reservedTokenSymbol,
+        }} on ${takeChainId} and earn ${
+          giveAmountWithoutDecimals - reservedAmountWithoutDecimals
+        } ${reservedTokenSymbol}`;
+
+        await this.twitterNotification.notify(message, { logger });
```

Comment:
I think we must introduce local storage to track announced orders. Let's ask Yaroslav: may be he will launch a small RDS for this